crmd/remote_lrmd_ra.c

```diff
@@ -80,6 +80,10 @@ typedef struct remote_ra_data_s {
     enum remote_migration_status migrate_status;
 
     gboolean active;
+    gboolean is_maintenance; /* kind of complex to determine from crmd-context
+                              * so we have it signalled back with the
+                              * transition from pengine
+                              */
```

Comment:
It suddenly occurs to me that, since it's coming from pengine, it's only valid on the DC. Am I missing how the connection host is learning it?